### PR TITLE
#165186130 Create client and stylist profile

### PR DESCRIPTION
**Pull request by kimbugp**
### What does this PR do?
- add the profiles and resolvers
#### Description of Task to be completed?
- add profile model
- add profiles schemas and resolvers
#### How should this be manually tested?

- Checkout to the branch `git checkout ch-create-user-profile-165186130`

- Make migrations `python manage.py makemigrations`

- Then migrate `python manage.py migrate`

#### What are the relevant pivotal tracker stories?
[#165186130](https://www.pivotaltracker.com/story/show/165186130)

### CODE REVIEW

File: apps/profile/apps.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class ServicesConfig(AppConfig):
```
Comment (RonKbS) [inline apps/profile/apps.py:4]: What's the use of this?

File: apps/profile/migrations/0001_initial.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    operations = [
+        migrations.CreateModel(
+            name='ClientProfile',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created_at', models.DateTimeField(auto_now_add=True)),
+                ('updated_at', models.DateTimeField(auto_now=True)),
+                ('first_name', models.CharField(blank=True, max_length=100, null=True)),
+                ('last_name', models.CharField(blank=True, max_length=100, null=True)),
+                ('sex', models.CharField(blank=True, max_length=100, null=True)),
```
Comment (RonKbS) [inline apps/profile/migrations/0001_initial.py:25]: Did we do the same data breach course? 😆

File: apps/profile/apps.py (hunk 1 above)
Comment (kimbugp) [reply apps/profile/apps.py:4]: It helps us add the profiles applications for introspection

File: apps/profile/schema.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    class Meta:
+        model = StylistProfile
+
+
+class Query(graphene.ObjectType):
+    client_profiles = graphene.List(ClientProfileModel)
+    stylist_profiles = graphene.List(StylistProfileModel)
+
+    def resolve_stylist_profiles(self, info, **kwargs):
+        return StylistProfile.objects.all()
```
Comment (KicaRonaldOkello) [inline apps/profile/schema.py:24]: Can we have an option of returning only one profile? Also for the client

**Review by KicaRonaldOkello — CHANGES_REQUESTED** [review]

File: apps/profile/schema.py (hunk 3 above)
Comment (kimbugp) [reply apps/profile/schema.py:24]: @KicaRonaldOkello, i have implemented this

**Review by KicaRonaldOkello — DISMISSED** [review]

**Review by KicaRonaldOkello — APPROVED** [review]

**Pull request merged**